lib/events.php

```diff
@@ -225,3 +225,40 @@ function widget_manager_update_group_event_handler($event, $object_type, $object
 		}
 	}
 }
+
+/**
+ * Remove fixed widgets from dashboard when parent widget is deleted from
+ * default widgets editor
+ *
+ * @param string     $event  "delete"
+ * @param string     $type   "object"
+ * @param ElggWidget $entity Widget
+ * @return void
+ */
+function widget_manager_remove_fixed_widgets($event, $type, $entity) {
```

Comment:
Do we really want to delete the fixed widgets? Why not 'release' them? That way users can keep using the widget and only remove if they no longer need it